azext_iot/tests/digitaltwins/test_dt_model_lifecycle_int.py

```diff
@@ -119,16 +119,9 @@ def test_dt_models(self):
             "dt model list -n {}".format(instance_name)
         ).get_output_in_json()
 
-        # Delete non-referenced models first
```

Comment:
We still need to exercise singular delete model in the integration tests

Reply:
I added a singular delete with a create after it.